irksome/tools.py

```diff
@@ -117,3 +118,43 @@ def __init__(self, msh):
 
     def Constant(self, val=0.0):
         return Function(self.V).assign(val)
+
+
+def ConstantOrZero(x, MC):
+    return Zero() if abs(complex(x)) < 1.e-10 else MC.Constant(x)
+
+
+def bc2space(bc, V):
```

Comment:
I agree this is a useful function to add, but we don't seem to use it anywhere yet...

Reply:
Right, I had thought I needed it at one point, hadn't removed.